internal/resolution/variables/bundle.go

```diff
@@ -26,17 +26,17 @@ func (b *BundleVariable) Dependencies() []*catalogmetadata.Bundle {
 	return b.dependencies
 }
 
-func NewBundleVariable(id deppy.Identifier, bundle *catalogmetadata.Bundle, dependencies []*catalogmetadata.Bundle) *BundleVariable {
-	var dependencyIDs []deppy.Identifier
+func NewBundleVariable(bundle *catalogmetadata.Bundle, dependencies []*catalogmetadata.Bundle) *BundleVariable {
+	dependencyIDs := make([]deppy.Identifier, 0, len(dependencies))
 	for _, bundle := range dependencies {
```

Comment:
Nit: unrelated to this PR, but we can we change the loop variable to be called `dependency` instead of `bundle` so that it doesn't shadow the `bundle` variable passed in as a parameter?

Reply:
I'll submit a follow up PR straight after merging this one.

Reply:
https://github.com/operator-framework/operator-controller/pull/455